ibis/expr/types/relations.py

```diff
@@ -2957,6 +2958,415 @@ def pivot_longer(
 
         return self.select(~pivot_sel, **new_cols)
 
+    @util.experimental
+    def pivot_wider(
+        self,
+        *,
+        id_cols: s.Selector | None = None,
+        names_from: str | Iterable[str | s.Selector] = "name",
+        names: Iterable[str] | None = None,
+        names_prefix: str = "",
+        names_sep: str = "_",
+        names_sort: bool = False,
+        values_from: str | Iterable[str | s.Selector] = "value",
+        values_fill: int | float | str | ir.Scalar | None = None,
```

Comment:
I think on `pivot_longer` we opted to drop this argument because it could just be followed by a `fillna` -- but I can see the utility here where there can be many columns which require filling.

Reply:
Selector support in `fillna` might obviate this a bit, but I still things it's common enough that having to spell the selectors for the values columns out twice is kind of a nuisance.